gse-util/src/main/java/com/powsybl/gse/util/SearchBar.java

```diff
@@ -219,6 +219,50 @@ public SearchBar(Searchable textArea) {
         });
     }
 
+    public void setSearchedArea(Searchable searchedArea) {
+        this.searchedArea = Objects.requireNonNull(searchedArea);
+    }
+
+    public CustomTextField getSearchField() {
+        return searchField;
+    }
+
+    public Button getDownButton() {
+        return downButton;
+    }
+
+    public Button getUpButton() {
+        return upButton;
+    }
+
+    public SearchMatcher getMatcher() {
+        return matcher;
+    }
```

Comment:
There is an encapsulation issue here: don’t give access to all members but provide business method

Reply:
updated